crates/ruff/src/message/gitlab.rs

```diff
@@ -116,8 +121,7 @@ fn fingerprint(
     let mut hasher = DefaultHasher::new();
 
     kind.rule().hash(&mut hasher);
```

Comment:
This is unrelated to your change but I noticed it now when reviewing this PR. It would probably be better to hash `kind.name` rather than the rule because the Rule's hash implementation changes every time a new rule is inserted into the `Rule` enum above which happens frequently. What do you think @zanieb ?

```suggestion
    kind.name.hash(&mut hasher);    
```

Reply:
This made sense to change, as you present  it, since  that could lead to as much a breaking change every time a new rule is added, as changing the hashing as we are doing with this.